Exercise_2.java

```diff
@@ -9,30 +20,62 @@
         StackNode(int data) 
         { 
             //Constructor here 
+            this.data = data;
         } 
     } 
     
 	
     public boolean isEmpty() 
     { 
         //Write your code here for the condition if stack is empty. 
+        if(root == null)
+            return false;
+        return true; 
     } 

```

Comment:
Please review this method 
If root is null, that implies that the list is empty and should return true